### PR TITLE
Created blog post about Proxy

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: _posts/blog/2019-12-16-accumulo-proxy.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+```
+Executing post run plugin: accumulo-proxy
+Installing Accumulo Proxy at /path/to/fluo-uno/install/accumulo-proxy-2.0.0-SNAPSHOT
+Accumulo Proxy 2.0.0-SNAPSHOT is running
+    * view logs at /path/to/fluo-uno/install/logs/accumulo-proxy/
+```
+
+Next, follow the instructions below to create a Python client that creates an Accumulo table
+named `pythontest` and writes data to it:
```
Comment (milleruntime) [inline _posts/blog/2019-12-16-accumulo-proxy.md]: Could mention which python version is required for using python.

Comment (mikewalch) [reply _posts/blog/2019-12-16-accumulo-proxy.md]: I added the version. I also created an issue to have it work with Python 3.0. https://github.com/apache/accumulo-proxy/issues/13

**Comment by ctubbsii** [comment]
Bundler installation is failing in Travis due to an update from Ruby 2.5 to 2.6, which comes with bundler installed as a default gem. I'm fixing that, but it shouldn't affect this PR... feel free to merge this whenever you're ready.

**Comment by ctubbsii** [comment]
I added a trivial commit to capitalize the words in the title to force Travis to retry the build, which was previously failing. Also, I added the word "Programming", so it's clear that it's clear the post is about other programming languages, and not clients written with non-English UIs.

**Pull request merged**